src/ASTBuilder.js

```diff
@@ -1,4 +1,5 @@
 const antlr4 = require('./antlr4/index')
+const ParseComments = require('./ParseNatSpec');
```

Comment:
Rename this file `natspec.js` and lowercase the parseComments function name please.